charts/enterprise-logs/README.md

```diff
@@ -0,0 +1,112 @@
+# Grafana Enterprise Metrics Helm Chart
```

Comment:
Since this is a new chart, we should ditch the manual README and adopt the gotmpl style used throughout the rest of the repository. For example, in loki-distributed https://github.com/grafana/helm-charts/blob/main/charts/loki-distributed/README.md.gotmpl